test/test_ops.py

```diff
@@ -1127,6 +1127,8 @@ def _perform_box_operation(self, box: Tensor, run_as_script: bool = False) -> Te
     def _run_test(self, test_input: List, dtypes: List[torch.dtype], tolerance: float, expected: List) -> None:
         def assert_close(box: Tensor, expected: Tensor, tolerance):
             out = self._perform_box_operation(box)
+            print("The computed box is: ", out)
```

Comment:
can we remove printing here? We don't usually print in tests.

Reply:
Oh yes, my bad, I used it to debug. Will remove, thanks for catching this.